src/Features/CSharp/Portable/InlineDeclaration/CSharpInlineDeclarationCodeFixProvider.cs

```diff
@@ -282,10 +299,15 @@ private static IEnumerable<SyntaxTrivia> MassageTrivia(IEnumerable<SyntaxTrivia>
                     return true;
                 }
 
+                var updatedTopmostContainer = topmostContainer;
                 var annotation = new SyntaxAnnotation();
-                var updatedTopmostContainer = topmostContainer.ReplaceNode(
-                    invocationOrCreation, invocationOrCreation.ReplaceNode(identifier, declarationExpression)
+
+                foreach ((var _, var declaration, var identifier) in declarationsToAdd)
+                {
+                    updatedTopmostContainer = updatedTopmostContainer.ReplaceNode(
```

Comment:
>ReplaceNode [](start = 70, length = 11)

You can use [ReplaceNodes](http://source.roslyn.io/#Microsoft.CodeAnalysis/Syntax/SyntaxNodeExtensions.cs,54) to replace multiple nodes.